### PR TITLE
Dockerized AWS CLI

### PR DESCRIPTION
**Pull request by papezt**

### CODE REVIEW

**Comment by underyx** [comment]
Please add it to the readme, like all the others 😇

File: aws-cli/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM python:3.7-alpine
```
Comment (aexvir) [inline aws-cli/Dockerfile]: Pin also the alpine version

Comment (aexvir) [reply aws-cli/Dockerfile]: Also I'd add yourself as maintainer so we know that you did it and probably you will be the one indicated to look into it if there are issues

Comment (underyx) [reply aws-cli/Dockerfile]: recently I started not pinning alpine, there's no major differences between minor versions and the official images updated `alpine` to point to the latest alpine version again

**Comment by aexvir** [comment]
I still see two commits, @underyx

**Pull request merged**